### PR TITLE
Fixes #27268 - Expose ansible collection endpoints

### PR DESCRIPTION
**Pull request by sjha4**

### CODE REVIEW

**Comment by jturel** [comment]
[test katello]

File: app/views/katello/api/v2/ansible_collections/base.json.rabl [hunk 1]
```diff
@@ -0,0 +1,8 @@
+object @resource
+
+attributes :id
+attributes :name
+attributes :namespace
+attributes :version
+attributes :checksum
+attributes :pulp_id
```
Comment (jlsherrill) [inline app/views/katello/api/v2/ansible_collections/base.json.rabl]: i know we do this for other types, but i wonder if we should stop providing a pulp_id, i know i'd like to see us do that long term, maybe we can go ahead and start with ansible.

**Comment by jlsherrill** [comment]
Overall looks good and seems to work fine, one small comment and then its good!

File: app/views/katello/api/v2/ansible_collections/base.json.rabl (hunk 1 above)
Comment (sjha4) [reply app/views/katello/api/v2/ansible_collections/base.json.rabl]: Removed pulp_id from rabl.

**Pull request merged**